openmc/plotter.py

```diff
@@ -201,23 +178,35 @@ def plot_xs(this, types, divisor_types=None, temperature=294., data_type=None,
         ax.set_xlim(_MIN_E, _MAX_E)
     else:
         ax.set_xlim(E[-1], E[0])
+
+    if isinstance(this, str):
+        # first entry in ELEMENT_SYMBOL is a neutron, the 1 removes this entry
+        if this in list(openmc.data.ELEMENT_SYMBOL.values())[1:]:
+            this = openmc.Element(this)
+        else:
+            this = openmc.Nuclide(this)
```

Comment:
Again, these objects are really meant to be deprecated. Can you refactor to not use these classes?